src/structure.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable linebreak-style */
+
+const structure = () => {
+  const container = document.querySelector('.container');
+
+  container.innerHTML = `<div class="row">
+  <div class="col-12 mt-5">
```

Comment:
**Changes Requested**
- [ ] The html is hard coded here. Let us use JavaScript to dynamically create the html on the `DOM`. 
Let us try to make every HTML element in the DOM dynamically generated.
Here is an example code snippet on how you could generate some html using JavaScript.

```
 const bookContainer = document.createElement('div');
    bookContainer.setAttribute('class', 'bookContainer');
    bookContainer.setAttribute('id', '');
    bookShelf.appendChild(bookContainer);

    const bookTitle = document.createElement('p');
    bookTitle.textContent = book.title;

    const bookPages = document.createElement('p');
    bookPages.textContent = book.pages;

    const bookAuthor = document.createElement('p');
    bookAuthor.textContent = book.author;

    bookContainer.appendChild(bookTitle);
    bookContainer.appendChild(bookPages);
    bookContainer.appendChild(bookAuthor);
    ```

Reply:
Thank you for your correction, the requested changes have already been implemented